website/docs/patterns/disabled-patterns/partials/guidelines/overview.md

```diff
@@ -0,0 +1,6 @@
+We recommend avoiding the use of disabled elements. These elements cannot be navigated to or interacted with, making them inaccessible to users who rely on assistive technology. Additionally, user research has validated our recommend approach through user feedback and direct observation.
```

Comment:
```suggestion
We recommend avoiding use of disabled elements. Disabled elements cannot be navigated to or interacted with, making them inaccessible to users who rely on assistive technology. User research and feedback has also validated this recommended approach.
```

Reply:
I think we should explain how research has validated this approach. What was the related feedback or observations?

Reply:
We kept it vague because there was no specific thing we can point at. This was feedback that came from Melina which is why we chose to phrase it that way.

Reply:
Updating phrasing to what you suggested and resolving.